SECURITY.md

```diff
@@ -0,0 +1,35 @@
+# Security Policy
+
+## Supported Versions
+
+Only vulnerabilities in the most recently released version and in the version currently on the master branch are considered to be significant.
+If you discover a vulnerability in an earlier version that may still be present in the most recently released version, you can still report it.
+
+## Supported Configuration
+
+Only vulnerabilities that apply to the application running in `production` mode are considered significant.
+
+## Reporting a Vulnerability
+
+If you would like to report a vulnerability, mention this to [web@gewis.nl](mailto:web@gewis.nl).
```

Comment:
The main reason I didn't include a PGP key yet is because I think it is difficult to share a PGP key amongst several members while we are constantly getting new members and having others depart. I think once we are contacted, we can share a temporary PGP key if necessary.

Reply:
Well, my suggestion includes having a smaller list of people dealing with security issues anyway. Its not needed to share issues with the entire web, and actually its better to have this be a small and more stable group.